README.md

```diff
@@ -68,29 +72,56 @@ More at https://github.com/lvmalware
 
 ```
 
-The general usage options are `-a` (to provide the authorization token), followed by the type of proxy and the worker's address.
+The general usage options are `-a` (to provide the authorization token), followed by the type of proxy and the 
+worker's address.
+
+For example, lets suppose your worker instance has the address `myinstance.workers.dev`, with the auth token of 
+`secret` (your CF token) and you want to run a SOCKS5 proxy server on port `1080` (default for SOCKS). 
+This could be done with the following command:
 
-For example, lets suppose your worker instance has the address `myinstance.workers.dev`, with the auth token of `secret` and you want to run a SOCKS5 proxy server on port `1080` (default for SOCKS). This could be done with the following command:
+> Make sure that you are in the root project directory.
 
 ```bash
+cd ..
 bun proxy.js -a secret -p 1080 socks myinstance.workers.dev
 ```
 
-Then configure your browser to use `127.0.0.1:1080` as a SOCKS5 proxy and enjoy the automatic ip address rotation :wink:.
+Then configure your browser to use `127.0.0.1:1080` as a SOCKS5 proxy and enjoy the automatic ip address 
+rotation :wink:.
 
 For a http proxy, just change the type from `socks` to `http`, for example:
 
 ```bash
 bun proxy.js -a secret -p 8080 http myinstance.workers.dev
 ```
 
+> In order to confirm that your IP has changed. 
+> You can easily see your request information on the CLI with:
+
+```bash
+curl -x localhost:8080 https://myip.wtf/json
+```
+
+> Or simply go to the address below on the URL bar.
```

Comment:
You should mention that the proxy should be configured on the browser beforehand, otherwise no change on the IP will be noticed.